### ISSUE TITLE
New labels in all repos

### ISSUE THREAD

**Issue by nhynes** [oasisprotocol/oasis-core#786]
For visibility, it'd be nice to have more labels that encourage people to take action on an issue.
A concrete example is in RFCs: a [WIP] PR should mean that the PR is not ready for review but there's no convention for those that _are_ ready for comments.
The proposed fix is to follow the example of Rust or TVM and add labels like
* "status: waiting on author"
* "status: ready for review" 
* "status: ready to merge"
